### DISCUSSION TITLE
Airplay does not work properly with last versions

### DISCUSSION THREAD

**Discussion by sergiosafer** [music-assistant/support#1998] [Q&A]
I have several Apple Airport (Gen1 and Gen2) connected to Home Assitant and everything worked perfectly until 2.0.0 BETA 93x but since BETA100 it does not recognize the Airport Gen1 and if I install the new version that you have recently released not only I can not see these devices, but also does not play on the Gen2.  
Thanks

**Comment by sergiosafer** [comment]
Here you are an example:
HA has detected 2 devices but MA can see only one

![image](https://github.com/music-assistant/hass-music-assistant/assets/132413705/4c9b8ce1-f9ab-40cb-8c4a-9556c79a8f0c)

![image](https://github.com/music-assistant/hass-music-assistant/assets/132413705/7ab81c91-bd61-411f-8b8f-59dce66c00ae)

**Reply by marcelveldt** [reply]
See https://github.com/music-assistant/hass-music-assistant/issues/1954

**Reply by sergiosafer** [reply]
Good thank you. I will follow this case.
The solution has not yet been found correct?

**Reply by OzGav** [reply]
This should be fixed with the latest betas

**Reply by kanesbit** [reply]
I'm on Current version: 2.0.0b119, and I too am having Airplay issues.  All devices are detected, but I cannot play to them.  Chromecast is fine.

Happy to do some debugging on my setup.  Please let me know

**Reply by marcelveldt** [reply]
> I'm on Current version: 2.0.0b119, and I too am having Airplay issues.  All devices are detected, but I cannot play to them.  Chromecast is fine.
> 
> 
> 
> Happy to do some debugging on my setup.  Please let me know

What sort of devices?
Note that some devices require enabling of encryption in the settings (of the player)

**Reply by kanesbit** [reply]
There are 3 devices
Marantz SR 7012
2x Marantz NR 1608

Confirming they've worked in the past

**Reply by kanesbit** [reply]
OK - AirPlay can eat a dick.  I'm using the HomeAssistant MediaPlayers and it's working

**Reply by marcelveldt** [reply]
Did you try to enable the encryption setting in the player settings of the AirPlay player?

**Reply by kanesbit** [reply]
The Marantz gear does not offer that option.  It can have an airplay password, which I've tried with and without

**Reply by OzGav** [reply]
It a MA setting for the player not on the device
